_posts/en/newsletters/2024-08-09-newsletter.md

```diff
@@ -0,0 +1,391 @@
+  cpfp carve out] relay rule, for LN commitment transactions.
+
+- [Bitcoin Core #29775][] adds a `testnet4` configuration option that will set
+  the network to [testnet4][topic testnet] as specified in [BIP94][].  Testnet4
+  includes fixes several problems with the previous testnet3 (see [Newsletter
+  #306][news306 testnet]).  The existing Bitcoin Core `testnet` configuration
+  option that uses testnet3 remains available but is expected to be deprecated
+  and removed in subsequent releases.
+
+- [Core Lightning #7476][] catches up to the latest [BOLT12
+  specification][bolt12 spec] updates by adding the rejection of zero-length
+  [blinded paths][topic rv routing] in [offers][topic offers] and invoice
+  requests. Additionally, it allows `offer_issuer_id` to be missing in offers
+  with a provided blinded path. In such cases, the key used to sign the invoice
+  is used as the final blinded path key, since it's safe to assume that the
+  offer issuer has access to this key.
+
+- [Eclair #2884][] implements [BLIP0004][blips endorsement] for [HTLC
+  endorsement][topic htlc endorsement], becoming the first LN implementation to
+  do so, to partially mitigate [channel jamming attacks][topic channel jamming
+  attacks] on the network. This PR enables the relaying of incoming endorsement
+  values, introducing a reputation system for LN where relaying nodes can choose
+  to pass an endorsement when forwarding an [HTLC][topic htlc] to the next hop.
+  If widely adopted by the network, endorsed HTLCs could receive preferential
+  access to scarce network resources such as liquidity and HTLC slots. This
+  implementation builds on previous Eclair work discussed in Newsletter
+  [#257][news257 eclair].
+
+
+- [LND #8952][] refactors the `channel` component in `lnwallet` to use the typed
+  `List`, as part of a series of PRs implementing dynamic commitments, also
+  known as [channel commitment upgrades][topic channel commitment upgrades].
+
+- [LND #8735][] adds the ability to generate invoices with [blinded paths][topic
+  rv routing] using the `-blind` flag in the `addinvoice` command. It also
+  allows payment of such invoices. Note that this is only implemented for BOLT11
+  invoices, as [BOLT12][topic offers] is not yet implemented in LND. [LND
+  #8764][] extends the previous PR by allowing the use of multiple blinded paths
+  when paying an invoice, specifically to perform multipath payments
+  ([MPP][topic multipath payments]).
+
+- [BIPs #1601][] merges [BIP94][] to introduce testnet4, a new version of
+  [testnet][topic testnet] that includes consensus rule improvements aimed at
+  preventing network attacks using only CPU mining. All previous soft forks are
+  enabled from the genesis block in testnet4, and the port used is `48333` by
+  default. See Newsletters [#306][news306 testnet4] and [#311][news311 testnet4]
+  for more details on how testnet4 fixes the issues that led to the fall of
```

Comment:
```suggestion
  for more details on how testnet4 fixes the issues that led to problematic behavior with
```

Reply:
The Fall of Testnet3 according to ChatGPT  :-)

![A_dramatic_and_heroic_battle_scene_depicting_'The_](https://github.com/user-attachments/assets/2faffc19-303a-4fbd-8f41-2861e543c051)